### PR TITLE
Fix broken path to zabbix repo download

### PR DESCRIPTION
**Pull request by DerMika**
I am currently unable to build our Zabbix install due to a 404 error when downloading the Zabbix RPM from the repo.

It seems like this folder was moved/renamed to `4.0` recently.

### CODE REVIEW

File: Dockerfile/dockbix-xxl-4.0/Dockerfile [hunk 1]
```diff
@@ -97,7 +97,7 @@ COPY container-files-zabbix /
 RUN \
   export FPING_VERSION=4.0 && \
   yum clean all && \
-  yum install -y https://repo.zabbix.com/zabbix/3.5/rhel/7/x86_64/zabbix-release-3.5-1.el7.noarch.rpm && \
+  yum install -y https://repo.zabbix.com/zabbix/4.0/rhel/7/x86_64/zabbix-release-3.5-1.el7.noarch.rpm && \
```
Comment (jangaraj) [inline Dockerfile/dockbix-xxl-4.0/Dockerfile]: Can we use https://repo.zabbix.com/zabbix/4.0/rhel/7/x86_64/zabbix-release-4.0-2.el7.noarch.rpm here?

Comment (DerMika) [reply Dockerfile/dockbix-xxl-4.0/Dockerfile]: I didn't want to change anything unnecessarily. I'll update the PR!

**Comment by DerMika** [comment]
Any idea when you can merge this and release a new tag on dockerhub?

**Comment by DerMika** [comment]
Can you let me know if this can be released to dockerhub? I still have a failing build right now because of this issue. 

Thanks for your time

**Comment by DerMika** [comment]
Thank you very much!

**Comment by jangaraj** [comment]
Sorry for the delay. Now all Docker hub 4.0 builds are green.

**Comment by DerMika** [comment]
No problem, glad it's fixed. I was able to redeploy my build, so I'm happy

**Pull request merged**